site/content/en/docs/Advanced/scheduling-and-autoscaling.md

```diff
@@ -110,70 +110,6 @@ Autoscaler to reduce the number of nodes in the cluster, [Controlling Disruption
 how to choose the `.eviction` setting appropriate for your `GameServer` or `Fleet`.
 {{% /feature %}}
```

Comment:
This feature block should be removed too since the enclosed content will be published with this release.

Reply:
removed the feature block in metrics.md.
site/content/en/docs/Reference/agones_crd_api_reference.html has 3000+ lines with the feature expiry version. do we need to delete this block? please confirm

Reply:
`site/content/en/docs/Reference/agones_crd_api_reference.html` should not be updated manually (there is a make target that updates it). Looking at the release checklist, it looks like we update it in the post-release PR so ignore that comment.